### PR TITLE
Removed unwanted/commented code, updated test cases.

### PR DESCRIPTION
**Pull request by AyushVerma2**
Removed unwanted/commented code, updated test cases.

### CODE REVIEW

**Comment by mikera** [comment]
Please sort out the build error

File: src/main/java/sg/dex/starfish/impl/url/RemoteHttpAsset.java [hunk 1]
```diff
@@ -53,17 +55,40 @@ private static URI getUri(String urlString) {
...
+    public static RemoteHttpAsset create( String uri,Map<String,Object> meta) {
+        return new RemoteHttpAsset(buildMetaData(uri, meta),  getUri(uri));
+    }
+
+	private static String buildMetaData(String uri, Map<String, Object> meta) {
+		String hash = Hex.toString(Hash.keccak256(uri));
+
+		Map<String, Object> ob = new HashMap<>();
+		ob.put(Constant.DATE_CREATED, Instant.now().toString());
+		ob.put(Constant.CONTENT_HASH, hash);
```
Comment (mikera) [inline src/main/java/sg/dex/starfish/impl/url/RemoteHttpAsset.java]: A hash of the URI is not the content hash!!!!!

This should not have got through testing. Please add test cases that validate content hashes properly.

File: src/test/java/sg/dex/starfish/integration/developerTC/AssetIdentity_02.java [hunk 2]
```diff
@@ -117,13 +116,9 @@ public void createAssetWithProvMetadata(){
         // uploading the Asset this remote Agent
         remoteAgent.uploadAsset(asset2);
 
-        // get the Remote asset ID which has been register using remote Agent
-        String assetID = remoteAsset.getAssetID();
 
         //verify prov info exists
         assertNotNull(remoteAsset.getMetadata().get("provenance").toString());
```
Comment (mikera) [inline src/test/java/sg/dex/starfish/integration/developerTC/AssetIdentity_02.java]: Are you actually testing the value of the provenance somewhere? Not null is a very weak test

File: src/test/java/sg/dex/starfish/integration/developerTC/MetaDataAccess_07.java [hunk 3]
```diff
@@ -66,7 +66,6 @@ public void testRemoteAssetMetaDataAsset() {
 
 
     private Map<String,Object> getMetaData(){
-        Map<String, Object> metaData = new HashMap<>();
         try {
             String METADATA_JSON_CONTENT = new String(Files.readAllBytes(Paths.get(METADATA_JSON_SAMPLE)));
             ObjectMapper objectMapper = new ObjectMapper();
```
Comment (mikera) [inline src/test/java/sg/dex/starfish/integration/developerTC/MetaDataAccess_07.java]: Don't use Jaskson!!! Use the JSON utilities already in starfish

File: src/test/java/sg/dex/starfish/integration/developerTC/AssetIdentity_02.java (hunk 2 above)
Comment (AyushVerma2) [reply src/test/java/sg/dex/starfish/integration/developerTC/AssetIdentity_02.java]: yes i am verifying in QueryProvenance_15.java

File: src/test/java/sg/dex/starfish/integration/developerTC/MetaDataAccess_07.java (hunk 3 above)
Comment (AyushVerma2) [reply src/test/java/sg/dex/starfish/integration/developerTC/MetaDataAccess_07.java]: done

**Comment by AyushVerma2** [comment]
looks good!!

**Pull request merged**